flink-clients/src/test/java/org/apache/flink/client/CliFrontendRunTest.java

```diff
@@ -98,6 +98,12 @@ public void testRun() {
 				RunOptions options = CliFrontendParser.parseRunCommand(parameters);
 				assertEquals("expectedSavepointPath", options.getSavepointPath());
 			}
+			// test configure configDir
+			{
+				String[] parameters = {"-D", "expectedConfigDirectory", getTestJarPath()};
```

Comment:
Both long and short option should be tested (though I think we can remove the short option).